README.md

```diff
@@ -21,6 +21,17 @@ $ npm install --save resin-register-device
 Documentation
 -------------
 
+Resin-Register-Device exports a factory function, which must be called with a dependencies object, containing a configured [`resin-request`](https://github.com/resin-io-modules/resin-request) instance.
+
+Example:
+```coffee
+deviceRegister = require('resin-register-device')({
+	request: require('resin-request')({
+		dataDirectory: '/opt/cache/resin'
```

Comment:
The first thing resin-request does it get a resin-token instance with this, which gets a resin-settings-storage instance, which creates this directory if it doesn't exist (and throws an exception if the parent `/opt/cache` directory doesn't exist).

After that it's unused though I think. Certainly, with a quick skim and some manual testing it looks like non-token usage only sends the api key in the query string, and skips the token and Authorization header completely.